ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Assertions.scala

```diff
@@ -50,9 +50,7 @@ object Assertions extends DiffExtensions {
         }
       // None both represents pattern that we do not care about as well as
       // exceptions that have no message.
-      case (GrpcException(GrpcStatus(`expectedCode`, Some(msg)), _), None) => ()
-      case (GrpcException(GrpcStatus(`expectedCode`, None), _), None) =>
-        ()
+      case (GrpcException(GrpcStatus(`expectedCode`, Some(_) | None), _), None) => ()
```

Comment:
Isn't `Some(_) | None` the same as `_`?

```suggestion
      case (GrpcException(GrpcStatus(`expectedCode`, _), _), None) => ()
```

Reply:
`null` matches neither 🙂 

Strictly speaking it doesn't preserve semantics, but it is probably what we meant anyway so I'll adopt that change.

Reply:
![You are technically correct. The best kind of correct.](https://user-images.githubusercontent.com/60356447/89566564-aa8ad600-d820-11ea-8a9f-959c051c7258.gif)